### PR TITLE
[f] work on linux

### PR DESCRIPTION
**Pull request by gmankab**
fixed work on linux

### CODE REVIEW

**Comment by gmankab** [comment]
@Exploited7, thanks

can you please also add a license? see https://github.com/Exploited7/outlook-account-creator/issues/20

**Comment by Exploited7** [comment]
Just did!

**Comment by gmankab** [comment]
@Exploited7, you did not merged commit from my pull request, so code is still broken on linux

```python
Traceback (most recent call last):
  File "/var/home/hmk/proj/outlook-account-creator/main.py", line 151, in <module>
    set_cmd_window_title(GENNED, LOCKED)
  File "/var/home/hmk/proj/outlook-account-creator/main.py", line 147, in set_cmd_window_title
    ctypes.windll.kernel32.SetConsoleTitleW(title)
    ^^^^^^^^^^^^^
AttributeError: module 'ctypes' has no attribute 'windll'
```

can you please merge it

File: main.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import random
 import string
 import json
+import platform
```
Comment (Exploited7) [inline main.py:13]: fixing linux enviroment

**Pull request closed without merging**